pkg/virt-handler/vm.go

```diff
@@ -2967,6 +2972,12 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 				d.recorder.Event(vmi, k8sv1.EventTypeWarning, "NicHotplug", err.Error())
 				errorTolerantFeaturesError = append(errorTolerantFeaturesError, err)
 			}
+
+			if err := d.hotUnplugVMIInterfaces(vmi); err != nil {
```

Comment:
As with the previous comment, is there a reason for implementing this extra function?
```suggestion
			if err := d.netConf.HotUnplugInterfaces(vmi); err != nil {
```